plugins/source/oracledb/resources/plugin/plugin_test.go

```diff
@@ -150,12 +163,12 @@ func TestPlugin(t *testing.T) {
 	if err := createTable(ctx, db, testTable); err != nil {
 		t.Fatal(err)
 	}
-	data := testdata.GenTestDataV1(testTable)
+	data := schema.GenTestData(testTable, schema.GenTestDataOptions{MaxRows: 1})
```

Comment:
what about null row (available if we gen 2 rows)?

Reply:
Tried it and currently blocked by https://github.com/cloudquery/plugin-sdk/pull/922

Reply:
https://github.com/cloudquery/cloudquery/pull/11115/commits/c9a80fd657ec924ebf1f9380c1874f12137d03b0 should test for null values and also properly support `number` schema type using `decimal`